sys-utils/hwclock-rtc.c

```diff
@@ -150,15 +132,23 @@ static int open_rtc_or_exit(const struct hwclock_control *ctl)
 static int do_rtc_read_ioctl(int rtc_fd, struct tm *tm)
 {
 	int rc = -1;
-
-	rc = ioctl(rtc_fd, RTC_RD_TIME, tm);
+	struct rtc_time rtc_tm;
```

Comment:
What about zero-initializing this?
I'm not sure if it's 100% necessary but it shouldn't hurt.

Reply:
I removed the `memset` from my patch because @karelzak initialized all the things in https://github.com/util-linux/util-linux/commit/0bc13e0175e645d625d9bcb81be7d55d6ffef189